server/terraform/project.tf

```diff
@@ -0,0 +1,41 @@
+variable "project_name" {
```

Comment:
Should this be `project_id` to match the convention for Google Cloud? Or is this a convention for Terraform?